CUSTOM_BUILDS.md

```diff
@@ -64,9 +64,12 @@ your CloudFormation or Terraform stack to use those new binaries.
 
 1. Build the code:
 
-   `make build` or `go build` should both work, the only difference is the
-   inclusion of versioning information into the binary, which may not matter
-   much anyway.
+   Run `make build` to generate a binary at `./AutoSpotting`. The resulting
```

Comment:
I removed the reference to `go build` since it's easier to only document one supported method to build binaries. I can add it back if you disagree.